pkg/clusteragent/admission/mutate/autoinstrumentation/lib_config_test.go

```diff
@@ -0,0 +1,89 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package autoinstrumentation
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/clusteragent/admission/mutate/common"
+)
+
+func TestInjectLibConfig(t *testing.T) {
```

Comment:
I pulled this test out of auto_instrumentation_test as is since it was testing things that are inside of the module.